platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -27,44 +27,24 @@ For Java, Groovy, Kotlin, and Android compatibility, see the [full compatibility
 
 <!-- Do not add breaking changes or deprecations here! Add them to the upgrade guide instead. -->
 
-<!--
-==========================================================
-ADD RELEASE FEATURES BELOW
-vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
-
-<a name="dependency-resolution"></a>
-### Dependency management improvements
-
-Gradle's [dependency management](userguide/core_dependency_management.html) infrastructure allows projects to depend on artifacts from external repositories, other projects from the same build, or projects from included builds.
-
-#### Root variant exposed by `ResolutionResult`
-
-The [`ResolutionResult` API](javadoc/org/gradle/api/artifacts/result/ResolutionResult.html), which contains the results of dependency resolution, now exposes the root variant of the resolved graph in addition to its owning component. 
-
-Previously, the API only exposed the root component, which exposed the first-level dependencies as well dependencies from other selected variants in the root component:
-
-```groovy
-def rootComponent = configurations.runtimeClasspath.incoming.resolutionResult.rootComponent
-```
+<a name="native-toolchains"></a>
+### Native toolchains support
```

Comment:
❓ I don't think this should be the headline of the release. Native support is not a focus area.

Reply:
I went back and forth on this one. Let's revisit this during our review.